bundles/org.openhab.core.io.rest.sitemap/src/main/java/org/openhab/core/io/rest/sitemap/internal/PageChangeListener.java

```diff
@@ -264,6 +261,13 @@ private SitemapWidgetEvent constructSitemapEventForWidget(Item item, State state
                 event.state = null;
             }
         }
+        String itemRGBHexCode = SitemapResource.getRGBHexCode(itemToBeSent);
+        String color = itemUIRegistry.getLabelColor(widget);
+        event.labelcolor = "itemValue".equals(color) && itemRGBHexCode != null ? itemRGBHexCode : color;
```

Comment:
What is the UI supposed to do when `itemValue` is returned? Shouldn't we return `null` in that case?

Reply:
Oh yes, that was my intention to never return "itemValue" and to rather return `null`.
There are 3 cases and only two are covered by this line.

Reply:
It is fixed.